config/locales/fr.yml

```diff
@@ -513,7 +513,7 @@ fr:
       public: Public
       public_long: Tout le monde peut voir vos statuts
       unlisted: Public sans être affiché sur le fil public
-      unlisted_long: Tout le monde peut voir vos statuts mais ils ne seront pas sur listés sur les fils publics
+      unlisted_long: Tout le monde peut voir vos statuts, mais ils ne seront pas sur listés sur les fils publics
```

Comment:
It's fine with or without the comma.

Reply:
This is to highlight the two proposals and mark the pause.

Reply:
There is no need for that from my point of view since this sentence is not difficult to understand as a whole.

I'm fine with or without it, and would recommend "without".